### PR TITLE
AC Automated Test - LRAC-9069, LRAC-9070, LRAC-9071, LRAC-9072, LRAC-9073 | master

### PR DESCRIPTION
**Pull request by LucasAcacio**
https://issues.liferay.com/browse/LRAC-9069
https://issues.liferay.com/browse/LRAC-9070
https://issues.liferay.com/browse/LRAC-9071
https://issues.liferay.com/browse/LRAC-9072
https://issues.liferay.com/browse/LRAC-9073

### CODE REVIEW

**Comment by LucasAcacio** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/InviteUsers.testcase [hunk 1]
```diff
@@ -30,48 +34,157 @@ definition {
...
 	test CanInviteMultipleUsers {
+		task ("go to user management and add multiple users") {
+			ACProperties.switchProperty(propertyName = "${propertyName}");
 
-		// TODO LRAC-9070 CanInviteMultipleUsers
+			ACNavigation.goToSettings();
 
+			ACSettings.goToUserManagement();
+
+			var emailList = "user1@liferay.com,user2@liferay.com,user3@liferay.com,user4@liferay.com";
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/InviteUsers.testcase:92]: nice @LucasAcacio !!

**Comment by alexcandidos** [comment]
Hi @LucasAcacio , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**